File: Ex-18/ex_18_python/Departamento.py
class Departamento:
    def __init__(self, nome=None, universidade=None):
        if nome is None:
            self.nome = ""
            self.universidade = None
        else:
            self.nome = nome
            self.universidade = universidade

    def setNome(self, nome):
        self.nome = nome

    def getNome(self):
        return self.nome

    def setUniversidade(self, universidade):
        self.universidade = universidade

    def getUniversidade(self):
        return self.universidade

File: Ex-20/ex_20_java/Departamento.java
public class Departamento {
    private String nome;
    private Universidade universidade;

    public Departamento() {
        this.nome = "";
        this.universidade = null;
    }

    public Departamento(String nome, Universidade universidade) {
        this.nome = nome;
        this.universidade = universidade;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public String getNome() {
        return this.nome;
    }

    public void setUniversidade(Universidade universidade) {
        this.universidade = universidade;
    }

    public Universidade getUniversidade() {
        return this.universidade;
    }
}
